docs/content/b2.md

```diff
@@ -466,6 +466,18 @@ If the custom endpoint rewrites the requests for authentication,
 e.g., in Cloudflare Workers, this header needs to be handled properly.
 Leave blank if you want to use the endpoint provided by Backblaze.
 
+The URL provided here SHOULD have the protocol and SHOULD NOT have
```

Comment:
This section is auto generated from b2.go so you can lose this chunk.

Reply:
Resolved